MantidPlot/src/ApplicationWindow.cpp

```diff
@@ -10478,17 +10478,17 @@ void ApplicationWindow::showStandAloneHelp() {
   QFile helpFile(helpPath);
   if (!helpPath.isEmpty() && !helpFile.exists()) {
     QMessageBox::critical(0, tr("MantidPlot - Help Files Not Found!"), // Mantid
-                          tr("The manual can be downloaded from the following "
+                          tr("The manual can be found at the following "
                              "internet address:") +
                               "<p><a href = "
-                              "http://soft.proindependent.com/"
-                              "manuals.html>http://soft.proindependent.com/"
-                              "manuals.html</a></p>");
+                              "http://www.mantidproject.org/"
+                              "MantidPlot:_Help>http://www.mantidproject.org/"
+                              "MantidPlot::_Help</a></p>");
```

Comment:
Sounds good, I've added a new issue #16326 to do this and look for other functions that aren't used any more.